test/models/bot/alegre_test.rb

```diff
@@ -79,12 +87,15 @@ def setup
         ]
       }.to_json)
       pm2 = create_project_media team: @pm.team, media: create_uploaded_image
-      assert_equal [pm1.id], Bot::Alegre.get_items_with_similar_image(pm2, 0.9)
+      response = {pm1.id => 0}
+      Bot::Alegre.stubs(:media_file_url).with(pm2).returns("some/path")
```

Comment:
Please make sure to `unstub` at the end of the test (or at once in `def teardown`) because it doesn't happen automatically in this version of the library. Something like: `Bot::Alegre.unstub(:request_api)`.